tests/tests/quota.sh

```diff
@@ -0,0 +1,134 @@
+
+TEST_UID=22222
+TEST_GID=44444
+
+# sys_setreuid() set fs[uid] to e[ug]id
+SET_UID="--ruid=$TEST_UID --euid=$TEST_UID"
+SET_GID="--rgid=$TEST_GID --egid=$TEST_GID --clear-groups"
+
+FILE="$T_D0/dir/file"
+
+sync_and_drop()
+{
+	sync
+	echo 1 > $(t_debugfs_path)/drop_weak_item_cache
+	echo 1 > $(t_debugfs_path)/drop_quota_check_cache
+}
+
+reset_all()
+{
+	rm -f "$FILE"
+	scoutfs quota-wipe -p "$T_M0"
+	getfattr --absolute-names -d -m - "$T_D0" | \
+		grep "^scoutfs.totl." | \
+		cut -d '=' -f 1 | \
+		xargs -n 1 -I'{}'  setfattr -x '{}' "$T_D0"
```

Comment:
on `el9` this croaks about `-n` and `-I` being mutually exclusive. I need to check whether my suspicion is right but I think that likely it has always been the case that `-I` implies `-n 1` anyway, even in old versions. So the fix will likely be that we just remove `-n 1` here.

Reply:
ok, just confirmed you can just remove `-n 1` and it'll work on `el7` as well as `el8` without issues as well.